src/secrets.js

```diff
@@ -180,4 +200,36 @@ function exportSecretToOutput(variableName, secretValue, exportSecretsToOutputs,
     }
 }
 
+function exportJsonFields(parsedJson, secretName, prefix, exportSecretsToOutputs, exportSecretsToEnvironment) {
+    if (prefix === undefined) {
+        prefix = convertPathNameToPrefix(secretName)
+    }
+    let outputName;
+    for (let key in parsedJson) {
+        outputName = prefix + "_" + key.toUpperCase()
+        exportSecretToOutput(outputName, parsedJson[key], exportSecretsToOutputs, exportSecretsToEnvironment)
+    }
+}
+
+function checkDuplicateKeys(parsedJson) {
+    const keyMap = new Map();
+
+    for (const [key, value] of Object.entries(parsedJson)) {
+        const upperCaseKey = key.toUpperCase();
+        if (keyMap.has(upperCaseKey)) {
+            throw new Error(`Duplicate key found in json: ${key}`);
+        }
+        keyMap.set(upperCaseKey, value);
+    }
+}
+
+function parseJson(jsonString) {
```

Comment:
I don't understand why hide the error here?

Reply:
requirements